frontend/src/utils/webapi.js

```diff
@@ -13,6 +16,13 @@ import Node from '../Models/node.js';
 var courseId = "42";
 var mainUrl = "";
 
+// TODO(nate): This is a hardcoded user id. change this dynamically during
+// runtime based on which user is actually viewing
+var userId = 1;
+
+//TODO: this is hardcoded. fix plz
+export var classId = "if44ov1fn5a505";
```

Comment:
Actually, scratch the last part. Forgot that `courseId` needs to be hardcoded for now. But we can definitely fetch `piazzaCourseId` from the backend. We should update `rootId` to use the backend API though, to fetch the list of roots and choose one by default (maybe just the first one in the list to start out).